tests/unit/routes/rentals/show-test.js

```diff
@@ -0,0 +1,24 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
I meant to ask about that. I'm used to using pretender and acceptance tests to test that the model is being returned correctly. It seemed out of the scope of this tutorial, however because acceptance tests are a whole other beast.

Thoughts on how to properly test this? Using mirage in the app means that we are essentially just testing that `queryRecord` creates the proper `GET` request for us (which of course it does).